test/index.js

```diff
@@ -293,6 +293,84 @@ describe('function', function() {
 
   });
 
+  describe('lift', function() {
+    it('is a binary function', function() {
+      eq(typeof S.lift, 'function');
+      eq(S.lift.length, 2);
+    });
+
+    it('type checks its arguments', function() {
+      assert.throws(function() { S.lift('wrong'); },
+                    errorEq(TypeError,
+                            '‘lift’ requires a value of type Function as its ' +
+                            'first argument; received "wrong"'));
+    });
+
+    it('lifts a function into the context of functors', function() {
+      eq(S.lift(R.multiply(2), S.Just(3)), S.Just(6));
+      eq(S.lift(R.multiply(2), S.Nothing()), S.Nothing());
+
+      eq(S.lift(R.multiply(2), S.Left(3)), S.Left(3));
+      eq(S.lift(R.multiply(2), S.Right(3)), S.Right(6));
+
+      eq(S.lift(R.multiply(2), [1, 2, 3]), [2, 4, 6]);
+      eq(S.lift(R.multiply(2), []), []);
+    });
+
+  });
+
+  describe('lift2', function() {
+    it('is a ternary function', function() {
+      eq(typeof S.lift2, 'function');
+      eq(S.lift2.length, 3);
+    });
+
+    it('type checks its arguments', function() {
+      assert.throws(function() { S.lift2('wrong'); },
+                    errorEq(TypeError,
+                            '‘lift2’ requires a value of type Function as its ' +
+                            'first argument; received "wrong"'));
+    });
+
+    it('lifts a function into the context of applicative functors', function() {
```

Comment:
Let's remove references to applicative functors. It's confusing to mix Fantasy Land and Haskell terminology.